react/features/toolbox/components/ToolboxV2.web.js

```diff
@@ -369,7 +372,9 @@ class ToolboxV2 extends Component<Props, State> {
                         && <OverflowMenuButton
                             isOpen = { this.state.showOverflowMenu }
                             onVisibilityChange = { this._onSetOverflowVisible }>
-                            <ul className = 'overflow-menu'>
+                            <ul
+                                aria-label = 'Overflow menu'
```

Comment:
why is this `aria-label` instead of `accessinilityLabel`?

Reply:
I see where there could be confusion. The abstractions for the HTML elements (the react components) that I use take in the prop/option "accessibilityLabel" but even they use "aria-label" when actually declaring the attribute on the HTML element, as the standard attribute for labeling an element for accessibility is "aria-label." Not sure how I should get around this as I wanted to be in parallel with react-native which does uses the "accessibilityLabel" prop. I could abstract this ul into a ReactComponent which takes in an accessibilityLabel prop/option, for example.